SU2_CFD/include/solver_structure.hpp

```diff
@@ -11033,290 +11116,6 @@ class CAdjTurbSolver : public CSolver {
   
 };
 
-/*! \class CPoissonSolver
```

Comment:
I think these solvers were removed a while ago, I removed their declaration from the header.